wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -978,28 +978,33 @@ def fragment(parser, token):
     Store a template fragment as a variable.
 
     Usage:
+
+    .. code-block:: html+django
+
         {% fragment as header_title %}
             {% blocktrans trimmed %}Welcome to the {{ site_name }} Wagtail CMS{% endblocktrans %}
         {% endfragment %}
+        {% include "my/custom/header.html" with title=header_title %}
```

Comment:
Add an `include` example so it's clearer why you might need the `fragment` over just adding `asvar` to `blocktrans`.

Edit: actually, now that I think about it, this use case can still be covered by `blocktrans trimmed asvar`...